lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -65,6 +65,14 @@ export const ClientAuthErrorMessage = {
         code: "nonce_not_found",
         desc: "nonce not found"
     },
+    authTimeNotFoundError: {
+        code: "auth_time_not_found",
+        desc: "Max Age was requested, and the ID token is missing the Auth Time variable (auth_time)"
```

Comment:
`auth_time` is an optional claim, that is not enabled by default. Can you add a note indicating that this optional claim must be enabled?

Reply:
And the aka.ms link for adding optional claims in your token?